tests/test_math/test_backend_manager.py

```diff
@@ -744,9 +744,9 @@ def test_displacement(self):
         Tests the ``displacement`` method.
         """
         cutoff = 5
-        alpha = 0.3 + 0.5 * 1j
+        alpha = 0.3 + 0.5j
         # This data is obtained by using qutip
-        # np.array(displace(40,alpha).data.todense())[0:5,0:5]
+        # np.array(displace(40,alpha).data.todense())[0:5,0:d5]
```

Comment:
Is the `d` here a typo?
```suggestion
        # np.array(displace(40,alpha).data.todense())[0:5,0:5]
```

Reply:
Yeah it is typo. But also qutip has changed since and it looks like that comment does not work anymore. Just updated that and added the qutip version.